Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Effectiveness.mo

```diff
@@ -1,108 +1,99 @@
     final displayUnit="degC")
     "Exhaust air temperature
-    " annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
-  Modelica.Blocks.Interfaces.RealInput v_flow_Sup(final unit="m3/s")
+    " annotation (Placement(transformation(extent={{-140,-100},{-100,-60}})));
+  Modelica.Blocks.Interfaces.RealInput VSup_flow(final unit="m3/s")
     "Volumetric flow rate of the supply air"
-    annotation (Placement(transformation(extent={{-140,-60},{-100,-20}})));
-  Modelica.Blocks.Interfaces.RealInput v_flow_Exh( final unit="m3/s")
-    "Volumetric flow rate of the exhaust air"
-    annotation (Placement(transformation(extent={{-140,-100},{-100,-60}})));
-  Modelica.Blocks.Interfaces.RealInput y(final unit="1") "Wheel speed ratio"
     annotation (Placement(transformation(extent={{-140,60},{-100,100}})));
-  Modelica.Blocks.Interfaces.RealOutput epsS(final unit="1")
+  Modelica.Blocks.Interfaces.RealInput VExh_flow( final unit="m3/s")
+    "Volumetric flow rate of the exhaust air"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}})));
+  Modelica.Blocks.Interfaces.RealInput wheSpe(final unit="1")
+    "Wheel speed ratio"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Interfaces.RealOutput epsSen(final unit="1")
     "Sensible heat exchanger effectiveness"
     annotation (Placement(transformation(extent={{100,30},{120,50}})));
-  Modelica.Blocks.Interfaces.RealOutput epsL(final unit="1")
-    "Latent heat exchanger effectivenessr"
+  Modelica.Blocks.Interfaces.RealOutput epsLat(final unit="1")
+    "Latent heat exchanger effectiveness"
     annotation (Placement(transformation(extent={{100,-50},{120,-30}})));
 
 protected
-   Real vRat
+   Real rat
    "Ratio of the average operating volumetric air flow rate to the nominal supply air flow rate";
-
-algorithm
-  // calculate effectiveness
-  if TSup > TExh then
-    // cooling mode
-    epsS :=y*(epsS_cool_partload + (epsS_cool_nominal - epsS_cool_partload)*(
-      vRat-0.75)/0.25);
-    epsL :=y*(epsL_cool_partload + (epsL_cool_nominal - epsL_cool_partload)*(
-      vRat-0.75)/0.25);
-  else
-    // heating mode
-    epsS :=y*(epsS_heat_partload + (epsS_heat_nominal - epsS_heat_partload)*(
-      vRat-0.75)/0.25);
-    epsL :=y*(epsL_heat_partload + (epsL_heat_nominal - epsL_heat_partload)*(
-      vRat-0.75)/0.25);
-  end if;
-  epsS := Buildings.Utilities.Math.Functions.smoothMax(
-       x1 = 0.01,
-       x2 = epsS,
-       deltaX = 1E-7);
-  epsS := Buildings.Utilities.Math.Functions.smoothMin(
-       x1 = 0.99,
-       x2 = epsS,
-       deltaX = 1E-7);
-
-  epsL := Buildings.Utilities.Math.Functions.smoothMax(
-       x1 = 0.01,
-       x2 = epsL,
-       deltaX = 1E-7);
-  epsL := Buildings.Utilities.Math.Functions.smoothMin(
-       x1 = 0.99,
-       x2 = epsL,
-       deltaX = 1E-7);
+   Real epsSen_ParLoa
+   "The partial load (75%) sensible heat exchanger effectiveness used for calculation";
```

Comment:
change to `epsSenPL` and in the comment change the `partial load` to `part load`.